containersQa/testlib.bash

```diff
@@ -930,40 +931,42 @@ EOF
 
 }
 
-function setAlgorithmTestsVars {
-  # Turning the tests off for now as there are random failure that are slowing testing for respins.
-  echo $SKIPPED3
-  exit
+function setupAlgorithmTesting {
+  # Skipping the FIPS tests if $OTOOL_CONTAINERQA_RUNCRYPTO is set to anything other than "true"
+  if [ "$OTOOL_CONTAINERQA_RUNCRYPTO" != "true"  ] ; then
+    echo "$SKIPPED8"
+    exit
+  fi
+
   checkAlgorithmsCode=`cat $LIBCQA_SCRIPT_DIR/CheckAlgorithms.java | sed -e "s/'//g"` # the ' characters are escaping and making problems, deleting them here
   cipherListCode=`cat $LIBCQA_SCRIPT_DIR/CipherList.java`
 }
 
+function isHostAndContainerCryptoIdentical() {
```

Comment:
Your method is asking a question you are not answering. Either add a check to the method or change the method name to logHostAndContainerCryptoPolicy()

Reply:
It is true, that now it is just logging. But on the longer run it will be asserting.  thus I vote to keep  isHostAndContainerCryptoIdentical.  However. the test itself is `..._isHostAndContainerCryptoIdentical` thus the method indeed can be `logHostAndContainerCryptoPolicy` and will jsut logging, and the new method, in far future will be `isHostAndContainerCryptoPolicy` and that one will be added, and will be asserting. So whether to rename or not, depends on future, whetehr the current method will be also asserting or not

Reply:
I changed the method name and added some more complex logging.